arcsi/api/utils.py

```diff
@@ -119,19 +119,19 @@ def broadcast_audio(
     return False
 
 
-def process(archive_base, archive_idx, archive_file, archive_name):
-    archive_file_name = form_filename(archive_file, archive_name)
+def save_file(archive_base, archive_idx, archive_file, archive_file_name):
```

Comment:
Does this code work for you? When I ran my first test upload it returned with an error on https://github.com/mmmnmnm/lahmacun_arcsi/blob/ead1b178daad65a1f05900e064754fe65620065f/arcsi/api/item.py#L189 because of TypeError. It's due to `archive_name` in the call vs `archive_file_name` in the signature, I guess, but maybe I'm wrong.

Reply:
@gammaw Sorry for the letdown on first try. I'm trying to retrace my steps, clearly something went wrong. I tested it extensively locally and on staging env too -- there are actually couple of hashed duplicates in our storage & db. I don't know why these argument keywords were missing. Did I change the param name in a later commit? I must have, because certainly the local state of apis wasn't saved. One quick remedy is that I've turned on autosave now. On dev I must have tried the uploads on a previous commit. An improvement here I'll make is to add the commit to the footer. Will create an issue for this later. 

I'd say it was more than an educated guess, it was the correct one. You are exactly right and I think that's a great thing. Thanks for the great debugging! I pushed a supposed fix.